cost-analyzer/templates/kubecost-cluster-controller-template.yaml

```diff
@@ -300,6 +300,9 @@ spec:
               name: {{ .Values.clusterController.secretName }}
               key: kubecostAPIKey
         {{- end }}
+        {{- if .Values.clusterController.extraEnv -}}
+        {{ toYaml .Values.clusterController.extraEnv | nindent 8 }}
+        {{- end }}
```

Comment:
Minor comment. I think it is stylistically better to make `extraEnv` the last item in the list of envs.